android/src/main/java/com/zoontek/rnpermissions/RNPermissionsModule.java

```diff
@@ -141,16 +143,13 @@ private boolean permissionExists(final String permission) {
 
   @ReactMethod
   public void checkNotifications(final Promise promise) {
-    final boolean enabled = NotificationManagerCompat
-      .from(getReactApplicationContext()).areNotificationsEnabled();
-
-    final WritableMap output = Arguments.createMap();
-    final WritableMap settings = Arguments.createMap();
-
-    output.putString("status", enabled ? GRANTED : BLOCKED);
-    output.putMap("settings", settings);
+    if(Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
```

Comment:
This forces everyone to update to `compileSdkVersion: 33`, even the user who don't care about this permission. As React Native template currently use `compileSdkVersion: 31`, this can be a bit annoying.